scene/animation/tween.cpp

```diff
@@ -713,6 +713,7 @@ void Tween::_tween_process(float p_delta) {
 
 		// Is the data not active or already finished? No need to go any further
 		if (!data.active || data.finish) {
+			all_finished = all_finished && (!data.active || data.finish);
```

Comment:
this line is useless : second part **&&...** is **true** by inclosing test ; then '**all_finished**' is not changed by this line of code ; better suppress it if logic is good (do not know)

Reply:
`all_finished = true;` is simpler and forces state as well

Reply:
https://github.com/godotengine/godot/pull/39801#discussion_r445088518